CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ MAJOR changes (breaking):
 
 MINOR changes (backwards-compatible):
 
-*
+* `ERROR`-level log lines are now logged to `stderr` in addition to `stdout`. This helps
+make errors more visible in the common case that `stdout` is redirected to a log file.
+This can currently be disabled via the (unstable) option `log-errors-to-stderr`.
```

Comment:
Interesting idea - maybe the option should be `log-errors-to-tty` in that case?

Reply:
That sounds good to me. I think this would mean we wouldn't need the option to disable the new behaviour for the tests, since neither stdout or stderr are a tty for the tests?

Reply:
done